examples/pytorch/mnist-ddp/mnist.ipynb

```diff
@@ -0,0 +1,627 @@
+      ]
+     },
+     "execution_count": 51,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "client.get_job_logs(job_name, follow=True)"
```

Comment:
Since get_job_logs returns value, you can do this to skip duplicated output:
```suggestion
    "_ = client.get_job_logs(job_name, follow=True)"
```